<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="dmarroquin" ignored="0" name="Au_Test" tests="11" failures="1" timestamp="02 ago 2017 22:21:31 GMT" time="217.071" errors="0">
  <testcase name="OpenUpload" time="27.066" classname="com.streame.tests.UploadMediaTest"/>
  <testcase name="UploadValidations" time="26.591" classname="com.streame.tests.UploadMediaTest"/>
  <testcase name="UploadMediaImage" time="14.327" classname="com.streame.tests.UploadMediaTest"/>
  <testcase name="UploadMediaAudio" time="15.85" classname="com.streame.tests.UploadMediaTest"/>
  <testcase name="UploadMediaVideo" time="19.806" classname="com.streame.tests.UploadMediaTest"/>
  <testcase name="UploadMediaDocument" time="17.664" classname="com.streame.tests.UploadMediaTest"/>
  <testcase name="UploadOnBackground" time="31.511" classname="com.streame.tests.UploadMediaTest"/>
  <testcase name="CancelInProgress" time="12.456" classname="com.streame.tests.UploadMediaTest"/>
  <testcase name="CancelCompleted" time="33.065" classname="com.streame.tests.UploadMediaTest"/>
  <testcase name="WrongType" time="2.873" classname="com.streame.tests.UploadMediaTest"/>
  <testcase name="SameName_Type" time="3.744" classname="com.streame.tests.UploadMediaTest">
    <failure type="java.lang.AssertionError" message="expected [A Image Of The Same Name Already Exists] but found [No
Yes]">
      <![CDATA[java.lang.AssertionError: expected [A Image Of The Same Name Already Exists] but found [No
Yes]
	at com.streame.tests.UploadMediaTest.SameName_Type(UploadMediaTest.java:527)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- SameName_Type -->
</testsuite> <!-- Au_Test -->
